Algorithms/Searching/Lower_Bound.cpp

```diff
@@ -0,0 +1,56 @@
+//Name : Shivam Singhal
+//Email ID : shivamsinghal0610@gmail.com
+
+//program to find minimum position of element lesser than given value 
+//in sorted fashion
+
+#include <iostream>
+using namespace std;
+
+void lower_bound(long arr[], long n, long x){
+	long count = (n-1), step, low = 0;
+	while(count>0){
+		step = (count/2);
+		low+=step;
+		//checking for lower_bound condition
+		if(arr[low] < x){
+			low++;
+			count-=(step+1);
+		}
+		else
+			count=step;
+	}
+	printf("Lower_bound of %lld is at position: %lld\n",x,low);
+}
+
+void printArray(long arr[], long n){
+	for(long i=0; i<n ;i++)
+		printf("%lld ",arr[i]);
+	printf("\n");
+}
+
+int main(){
+	long n;
```

Comment:
Here declaration should be `long long int` as you have used `%lld` in your input statement. Please fix it complete code

Reply:
Yes, done.